include/circt/Dialect/Arc/ArcPasses.td

```diff
@@ -62,6 +62,16 @@ def FindInitialVectors : Pass<"arc-find-initial-vectors", "mlir::ModuleOp"> {
   let summary = "Find initial groups of vectorizable ops";
   let constructor = "circt::arc::createFindInitialVectorsPass()";
   let dependentDialects = ["arc::ArcDialect"];
+  let statistics = [
+    Statistic<"numOfVectorizedOps", "vectorOps",
+      "Total number of ops that were vectorized">,
+    Statistic<"numOfSavedOps", "numOfSavedOps",
+      "Total number of ops saved after FindInitialVectors pass">,
```

Comment:
Isn't this always equivalent to `vectorOps - numOfVectorsCreated` and could, therefore, be removed?

Reply:
```
a = b + c
d = e + f
g = h + i

after vectorization: [a, d, g] = [b, e, h] + [c, f, i]

biggestSeedVector: 3
numOfSavedOps: 2 (we had 3 ops but now we have one)
numOfVectorsCreated: 1
numOfVectorizedOps: 3
```

Reply:
Exactly, 3-1=2. I don't mind having it, just wanted to point out the redundancy.